mbedtls/src/self_test.rs

```diff
@@ -10,18 +10,18 @@
 //!
 //! Calling mbedTLS self-test functions before they're enabled using the
 //! `enable()` function here will result in a panic.
-#[cfg(any(target_os = "none", not(feature = "std")))]
+#[cfg(any(target_os = "none", not(feature = "std"), target_env = "sgx"))]
```

Comment:
please order so that all `target_` checks are next to each other.

Reply:
Thanks. Done